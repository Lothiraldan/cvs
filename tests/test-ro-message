#!/bin/sh
HG=hg
"$HG" init
mkdir b
echo 'Bouncy' >b/bouncy
echo 'tricycle' >b/vehicle
"$HG" add b/bouncy
"$HG" add b/vehicle
"$HG" commit -m 'Adding bouncy'
echo 'bouncy' >>b/bouncy
"$HG" commit -m 'Making it bouncier'
"$HG" update -C 0
echo 'stationary' >>b/vehicle
"$HG" commit -m 'Clarifying the vehicle.'
"$HG" update -C 1
chmod a-w b/vehicle
"$HG" update -m 2 2>&1 | sed 's|^\(.*[ 	]\)/tmp/[^/]*/\(.*\)$|\1\2|g'
