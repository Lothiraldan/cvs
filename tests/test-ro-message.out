1 files updated, 0 files merged, 0 files removed, 0 files unresolved
2 files updated, 0 files merged, 0 files removed, 0 files unresolved
abort: Permission denied: test-ro-message/b/vehicle
